### ISSUE TITLE
'meson test' skips tests built for build machine thinking it's a cross compiled binary

### ISSUE THREAD

**Issue by aakre** [mesonbuild/meson#9226]
**Describe the bug**
This might not be a bug but just a misconfiguration on my end. Hoping for some guidance to get this sorted :) 

I have a cross compilation project where I also compile unit tests to be run on the build machine. My project builds and I can manually run my unit test binary on the build machine. **My issue:** If I run 'meson test' then all tests are skipped. In meson-logs/testlog.txt I find the error message: *Not run because can not execute cross compiled binaries.* My unit test binary is called test_event and I can run it on the build machine:
```
file test_event
test_event: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=4536c0dc4f09a1449d4f05aa02a28949305dbfcd, for GNU/Linux 3.2.0, with debug_info, not stripped

./test_event
[====] Synthesis: Tested: 7 | Passing: 7 | Failing: 0 | Crashing: 0
```

**To Reproduce**
I have this meson.build in the subdir with my tests:
```
test_tool = declare_dependency(
    dependencies: meson.get_compiler('c', native:true).find_library('criterion', dirs: meson.current_source_dir() + '/criterion-v2.3.3/lib'),
    include_directories: include_directories('criterion-v2.3.3/include'),
)

e = executable('test_event',
    dependencies: test_tool,
    sources: ['event/test_event_package.c', '../cloud/src/event_package.c', '../cloud/cos-c/coswrite.c', '../cloud/cos-c/binaryio.c'],
    include_directories: include_directories('../'),
    native: true,
)

test('test_event_package', e)
```

**Expected behavior**
I expected 'meson test' to run my unit tests, recognising the test binary as built for the build machine

**system parameters**
* cross build
* build machine: pop_os 21.04
* python 3.9.5
* meson 0.56.2
* ninja 1.10.1

cheers!

**Comment by oyvindaakre** [comment]
bump :) 

tldr; How can I utilize meson test harness to run tests on the build machine in a cross build?

**Comment by aakre** [comment]
bump :)

tldr; How can I utilize meson test harness to run tests on the build machine in a cross build?

**Comment by jpakkane** [comment]
Does setting `needs_exe_wrapper = false` in the cross file's `[properties]` work?

**Comment by aakre** [comment]
yes it does! thank you so much :)
